tag.sh

```diff
@@ -52,7 +51,7 @@ if [ -z "${SKIP_ORIGIN_HEAD_CHECK:-}" ]; then
     fi
 fi
 
-git tag --sign -m "${TAG} - ${SUMMARY}" "${TAG}" "${TAG_COMMIT}"
+git tag --sign -m "Phylum CLI ${TAG}" "${TAG}" "${TAG_COMMIT}"
```

Comment:
I don't think anybody reads tag messages... But I'll take suggestions if there's something else I should put here.

Reply:
I think you are correct. This should be good enough.